agent-install/edgeNodeFiles.sh

```diff
@@ -220,13 +220,10 @@ function checkPrereqsAndInput () {
             echo "CLUSTER_URL is not set. Attempting to set it from Kubernetes resources..."
 
             NAMESPACE="$($K8S_CLI_TOOL get eamhub -A --no-headers | awk 'NR==1 {print $1}')"
-            CUSTOM_RESOURCE="$($K8S_CLI_TOOL get eamhub -n "$NAMESPACE" --no-headers | awk '{print $1}')"
-
-            if [[ -z "$NAMESPACE" || -z "$CUSTOM_RESOURCE" ]]; then
+            if [[ -z "$NAMESPACE" ]]; then
```

Comment:
a small nit... I would change the fatal message

fatal 1 "Error: Unable to determine namespace or custom resource."
            
            to
fatal 1 "Error: Unable to determine namespace."